frontend/glideinFrontendElement.py

```diff
@@ -982,6 +968,108 @@ def iterate_one(self):
 
         return
 
+    def get_scitoken(self, elementDescript, trust_domain):
+        """Look for a local SciToken specified for the trust domain.
+
+        Args:
+            elementDescript (ElementMergedDescript): element descript
+            trust_domain (string): trust domain for the element
+
+        Returns:
+            string, None: SciToken or None if not found
+        """
+
+        scitoken_fullpath = ""
+        cred_type_data = elementDescript.element_data.get("ProxyTypes")
+        trust_domain_data = elementDescript.element_data.get("ProxyTrustDomains")
+        if not cred_type_data:
+            cred_type_data = elementDescript.frontend_data.get("ProxyTypes")
+        if not trust_domain_data:
+            trust_domain_data = elementDescript.frontend_data.get("ProxyTrustDomains")
+        if trust_domain_data and cred_type_data:
+            cred_type_map = eval(cred_type_data)
+            trust_domain_map = eval(trust_domain_data)
+            for cfname in cred_type_map:
+                if cred_type_map[cfname] == "scitoken":
+                    if trust_domain_map[cfname] == trust_domain:
+                        scitoken_fullpath = cfname
+
+        if os.path.exists(scitoken_fullpath):
+            try:
+                logSupport.log.debug("found scitoken %s" % scitoken_fullpath)
```

Comment:
I see that logSupport is being used here to print the string. Does f-string formatting apply here like how we use it in python 3.x?